bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,246 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+import json
+
+
+@linter(executable='csscomb',
+        output_format='corrected',
+        use_stdin=True,
+        result_message='Change coding style based on CSS rules written in ' +
+                       '.csscomb.json')
```

Comment:
the user didn't necesssarily define the rules in csscomb.json, this info doesn't really belong in the message IMO